Gibe.DittoProcessors.Test/DittoProcessorTests/FakeRootNodeType.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Gibe.Navigation.Umbraco.NodeTypes;
+using Umbraco.Core.Models;
+
+namespace Gibe.DittoProcessors.Test.DittoProcessorTests
+{
+	internal class FakeRootNodeType : INodeType
```

Comment:
Shouldn't these Fakes go into the project they're faking so we don't have duplicate fakes everywhere we're testing stuff